packages/@aws-cdk/aws-lambda-golang/test/lambda-handler-vendor/vendor/github.com/aws/aws-lambda-go/LICENSE

```diff
@@ -0,0 +1,203 @@
+
```

Comment:
does all this vendor stuff need to be committed?

Reply:
> Nice work!
> 
> Don't know `go` enough to correctly review this but sharing my experience from `@aws-cdk/aws-lambda-nodejs`.
> 
> Wonder if the common logic/structure of `@aws-cdk/aws-lambda-nodejs` and `@aws-cdk/aws-lambda-golang` should be centralized somewhere...

I tried to follow the `aws-lambda-nodejs` structure as closely as possible, so I definitely think the common logic/structure could be centralized somewhere. It would also make adding additional languages much easier.

Reply:
removed the vendor from the tests.